README.md

```diff
@@ -98,7 +113,7 @@ double NewtonRaphson::find(double xin)
 } // namespace rootfinding
 ```
 
-A simple CLI program would look like
+Finally, we can write our simple CLI program. It would look like
```

Comment:
finally seems a bit optimistic at the start of the guide
```suggestion
We are now ready to call the algorithm in a simple CLI program. It would look like
```